Tasks/Common/artifacts-common/webapi.ts

```diff
@@ -0,0 +1,25 @@
+import * as api from 'azure-devops-node-api';
+import { IRequestOptions } from 'azure-devops-node-api/interfaces/common/VsoBaseInterfaces';
+import * as tl from 'azure-pipelines-task-lib/task';
+
+export function getWebApiWithProxy(serviceUri: string, accessToken: string): api.WebApi {
```

Comment:
Should we accept an optional 'options' parameter? For some use cases, for example where we return a redirect to a blobstore url in the content apis, we need to pass in additional options which specify to not forward auth headers (like the options defined here: https://github.com/microsoft/typed-rest-client/blob/master/lib/HttpClient.ts#L392). We can merge the users and our options before creating the client.

Reply:
Fine with me, feel free to change it as needed when you merge your changes with this.